mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -3694,7 +3691,14 @@ async fn check_balance_for_maker_orders(ctx: MmArc, ordermatch_ctx: &OrdermatchC
 
     for (uuid, order) in my_maker_orders {
         let order = order.lock().await;
+
         if order.available_amount() >= order.min_base_vol || order.has_ongoing_matches() {
+            if order.has_ongoing_matches() {
+                continue;
+            }
```

Comment:
`has_ongoing_matches` is OR-ed in the first if clause and then checked alone in the second.

we could move it out before the `available_amount()` check.